### DISCUSSION TITLE
taiko hitsounds are wrong on a specific map

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#16935] [Q&A] [answered]
https://osu.ppy.sh/beatmapsets/1412083#taiko/2910926

https://youtu.be/hlwjLBvFsT4 the hitsounds are normal at the start, but become wrong later on (I have custom hitsounds turned off)

**Comment by peppy** [comment, marked as answer]
Please explain what you are expecting to hear.

The beatmap uses drum samples after that point in time:

![osu! 2022-02-22 at 06 52 38](https://user-images.githubusercontent.com/191335/155078317-feea4766-d608-4913-850b-e5b0f5119f5e.png)

Which are correctly playing in the video you linked.

**Reply by chayleaf** [reply]
this is the first time i hear this sound, but it turns out it's the same in stable... resolving this
